server/services/address.service.js

```diff
@@ -56,9 +56,22 @@ module.exports = class AddressService {
   }
 
   static async _queryAddressEndpoint (postcode, pageNumber, pageSize) {
+    /**
```

Comment:
I would refactor lines59-70 into a separate method, maybe called _getCertificate or similar. Would make it a bit clearer what is going on.